### ISSUE TITLE
Crash Group::GetInvited

### ISSUE THREAD

**Issue by Idoitbetter** [TrinityCore/TrinityCore#26624]
**Description:**

Crash: https://gist.github.com/Idoitbetter/99b44c203626ccfb4cce63df7fa427a2
last line of server.log can appear things like this:
```
Attempt to get value from non-existing value field: 0 (count: 0) for object typeid: 32766 type mask: 35536
```
or
```
Attempt to get value from non-existing value field: 0 (count: 0) for object typeid: 0 type mask: 0
```

It is an old crash, I have been seeing it for years, there are old issues closed with the same crash
https://github.com/TrinityCore/TrinityCore/issues/21505
https://github.com/TrinityCore/TrinityCore/issues/15071
https://github.com/TrinityCore/TrinityCore/issues/17637

**Steps to reproduce the problem:**

I don't know

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

f418631de0c193772e24f379db5aafcc01f3fb6a

**Operating system:** 

Debian

**Comment by jackpoz** [comment]
Do you have any custom patch/code ?

**Comment by Idoitbetter** [comment]
no, I do not understand what can happen, I have tried to create groups, join bg, delete characters in the group, leave the group, but nothing
